installer/src/panels/com/jboss/devstudio/core/installer/bean/P2IUListBean.java

```diff
@@ -89,4 +88,25 @@ public String getCommaSeparatedLocationStringList() {
 		}
 		return allLocationsList.toString();
     }
+
+    /**
+     * 
+     * @return a comma-separated list of additional IU sizes (estimated)
+     */
+    public String getCommaSeparatedSizeStringList() {
```

Comment:
Please add test for this method to [P2IUListBeanTest.java](https://github.com/jbdevstudio/jbdevstudio-product/blob/master/installer/src/test/com/jboss/devstudio/core/installer/bean/P2IUListBeanTest.java).